README.md

```diff
@@ -236,23 +262,39 @@ If you are reading this it looks like you are looking to add an egg to your serv
 [Xonotic](game_eggs/xonotic/xonotic)
 
 ## [Monitoring](/monitoring/)
+
 ### Prometheus
+
 * [Prometheus](/monitoring/prometheus)
```

Comment:
Any of these from here down probably don't need to be indented and given the `* ` dots unless there are multiple that match

Reply:
I just added those for consistency